orchagent/portsorch.cpp

```diff
@@ -223,6 +224,25 @@ bool PortsOrch::setPortAdminStatus(sai_object_id_t id, bool up)
     return true;
 }
 
+bool PortsOrch::setPortMtu(sai_object_id_t id, sai_uint32_t mtu)
+{
+    SWSS_LOG_ENTER();
+
+    sai_attribute_t attr;
+    attr.id = SAI_PORT_ATTR_MTU;
+    /* mtu + 14 + 4 + 4 = 22 bytes */
+    attr.value.u32 = mtu + sizeof(struct ether_header) + FCS_LEN + VLAN_TAG_LEN;
```

Comment:
I agree with the fix. It is expected that port MTU at SAI needs to add eth header size.

---
In reply to: [117753359](https://github.com/Azure/sonic-swss/pull/218#discussion_r117753359) [](ancestors = 117753359)